### PR TITLE
Launch the Grafana dashboard from the Tendrl-UI

### PR DESCRIPTION
**Pull request by cloudbehl**
Tendrl-bug-id: https://github.com/Tendrl/ui/issues/571

### CODE REVIEW

File: src/commons/services/utils.js [hunk 1]
```diff
@@ -270,5 +270,33 @@
...
+                initialUrl,
+                dashboardTypes,
+                uri;
+
+            dashboardTypes = {
+                glance: "tendrl-gluster-at-a-glance",
+                volumes: "tendrl-gluster-volumes",
+                hosts: "tendrl-gluster-hosts",
+                bricks: "tendrl-gluster-bricks"
+            }
```
Comment (gnehapk) [inline src/commons/services/utils.js]: @cloudbehl Please add semicolon here.

File: src/commons/services/utils.js [hunk 2]
```diff
@@ -270,5 +270,33 @@
...
+            if(dashboardName === "glance"){
+                uri = initialUrl + "?var-cluster_id="+ grafanaObj.clusterId;
+            } else if(dashboardName === "volumes") {
+                uri = initialUrl + "?var-cluster_id="+ grafanaObj.clusterId + "&var-volume_name=" + grafanaObj.volumeName;
+            } else if(dashboardName === "hosts") {
+                uri = initialUrl + "?var-cluster_id="+ grafanaObj.clusterId + "&var-host_name=" + grafanaObj.hostName;
+            } else if(dashboardName === "bricks") {
+                uri = initialUrl + "?var-cluster_id="+ grafanaObj.clusterId + "&var-volume_name=" + grafanaObj.volumeName + "&var-host_name=" + grafanaObj.hostsName; + "&var-brick_path=" + grafanaObj.brickName;
+            }
+            window.open(uri);
```
Comment (gnehapk) [inline src/commons/services/utils.js:298]: @cloudbehl Is this opening the dashboard in a new tab?

File: src/modules/hosts/host-list/host-list.js [hunk 3]
```diff
@@ -46,6 +47,11 @@
             }, 1000 * config.nodeRefreshIntervalTime, 1);
         }
 
+        function redirectToGrafana(host, $event){
+            utils.redirectToGrafana("hosts", $event, {clusterId: vm.clusterId,
+                                                      hostName: host.name.split(".").join("_")});
+        }
```
Comment (gnehapk) [inline src/modules/hosts/host-list/host-list.js]: @cloudbehl I think its not correctly indented. Whole statement should be in one line.

File: src/modules/volumes/volume-list/volume-list.js [hunk 4]
```diff
@@ -59,6 +60,11 @@
             $interval.cancel(volumeTimer);
         });
 
+        function redirectToGrafana(volume, $event){
+            utils.redirectToGrafana("volumes", $event, {clusterId: vm.clusterId,
+                                                      volumeName: volume.name});
```
Comment (gnehapk) [inline src/modules/volumes/volume-list/volume-list.js]: @cloudbehl Same as my previous comment.

File: src/commons/services/utils.js (hunk 2 above)
Comment (cloudbehl) [reply src/commons/services/utils.js:298]: @gnehapk yes

**Review by a2batic — APPROVED** [review]

**Review by gnehapk — APPROVED** [review]

**Pull request merged**